Automattic/newspack-newsletters
# Add/posts inserter featured image pos

### All Submissions:






### Changes proposed in this Pull Request:



Adds featured image position option to Posts Inserter block, as well as the link from image to post. 

Closes #128 

### How to test the changes in this Pull Request:

0. Create or edit a newsletter
1. Insert a Posts Inserter block
2. Observe image alignment options in the block settings, if "Display featured image" option is set to true
3. Change the aligment – observe layout of the posts change accordingly
4. Test the alignment options in an email (after posts insertion)

### Other information:
